packages/block-library/src/video/tracks-editor.js

```diff
@@ -235,8 +240,21 @@ export default function TracksEditor( { tracks = [], onChange } ) {
 						/>
 					);
 				}
+
 				return (
 					<>
+						{ tracks.length === 0 && (
+							<div className="block-library-video-tracks-editor__tracks-informative-message">
+								<h2 className="block-library-video-tracks-editor__tracks-informative-message-title">
+									{ __( 'Text tracks' ) }
+								</h2>
+								<p className="block-library-video-tracks-editor__tracks-informative-message-description">
+									{ __(
+										'Tracks can be subtitles, captions, chapters, or descriptions. They help make your content more accessible to a wider range of users.'
+									) }
+								</p>
```

Comment:
In general these components might need a bit design refresh, but I think we should at least match trunk. That means we should add the separator that previously was added by the `MenuGroup`, no?

Reply:
That separator is part of the `components-menu-group` MenuGroup component. It is meant to separate menu groups.

Previously, in the initial state, there were two groups and one of them was used incorrectly. Now, there's only one group so there's nothing to separate. After uploading a track, there are two groups and the separator is still there. Screenshot:

![Screenshot 2024-11-18 at 14 22 40](https://github.com/user-attachments/assets/772cb002-ae42-4c77-940c-b66b386273a4)